support-frontend/app/controllers/Application.scala

```diff
@@ -39,7 +39,7 @@ case class PaymentMethodConfigs(
 class Application(
     actionRefiners: CustomActionBuilders,
     val assets: AssetsResolver,
-    testUsers: TestUserService,
+    testUserService: TestUserService,
```

Comment:
I renamed this as keeping things universally named helps (me) with discoverability.